framework/src/main/java/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -60,8 +60,38 @@
   /** A checker that contains command-line arguments and other information. */
   protected final SourceChecker checker;
 
+  /**
+   * A triple of field, value corresponding to the annotations on its declared type, value of its
+   * initializer. The value of the initializer is {@code null} if the field does not have one.
+   *
+   * @param <V> type of value
+   */
+  public static class FieldValues<V extends CFAbstractValue<V>> {
```

Comment:
I think this class represents only initial values, in which case I would rename it to indicate that, such as to `FieldIinitialValue`.
Since it's for only one field, I would make the name singular, to clarify some use sites (see comments elsewhere).